packages/app-store/office365calendar/lib/CalendarService.ts

```diff
@@ -73,6 +73,10 @@ export default class Office365CalendarService implements Calendar {
         body: JSON.stringify(this.translateEvent(event)),
       });
 
+      const responseJson = await handleErrorsJson(response);
+
+      return { ...responseJson, iCalUID: responseJson.iCalUId };
```

Comment:
Notice in Outlook the property has a lower case "d". Through out the call I think we should standardize it to UID as that's the property name in the .ics file